api/namex/constants/__init__.py

```diff
@@ -444,46 +451,52 @@ class LegacyEntityTypes:
+    ('BECR', EntityTypes.CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('ULCB', EntityTypes.CONTINUE_IN.value, RequestAction.CNV.value, True), # continued in Unlimited Liability Company -> continued in BC Limited Company
+    ('ULCB', EntityTypes.CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+
+    ('CCC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.CHG.value, True),
+    ('CCC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('RCC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.REST.value, True),
+    ('RCC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.REH.value),
+    ('RCC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.REN.value),
+    ('RCC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('CCV', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.CNV.value, True), # continued in BC Limited Company -> continued in Community Contribution Company
+    ('CCV', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('BECC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.CNV.value, True), # continued in Benefit Company -> continued in Community Contribution Company
+    ('BECC', EntityTypes.CCC_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+
+    ('CUL', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.CHG.value, True),
+    ('CUL', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.RESUBMIT.value, True),
+    ('RUL', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.REST.value, True),
+    ('RUL', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.REH.value),
+    ('RUL', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.REN.value),
+    ('RUL', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('UC', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.CNV.value, True), # continued in BC Limited Company -> continued in Unlimited Liability Company
+    ('UC', EntityTypes.ULC_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+
+    ('BEC', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.CHG.value, True),
+    ('BEC', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('BERE', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.REST.value, True),
+    ('BERE', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.REH.value),
+    ('BERE', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.REN.value),
+    ('BERE', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('BECV', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.CNV.value, True), # continued in BC Limited Company -> continued in Benefit Company
+    ('BECV', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('ULBE', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.CNV.value, True), # continued in Unlimited Liability Company -> continued in Benefit Company
+    ('ULBE', EntityTypes.BEN_CONTINUE_IN.value, RequestAction.RESUBMIT.value),
```

Comment:
This is now the complete list of all applicable NR types for all business types.

You can compare this with:
https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts